models/topics_models.js

```diff
@@ -11,6 +11,14 @@ function fetchEndPoints() {
     return require(`${__dirname}/../endpoints.json`)
 }
 
-module.exports = {fetchTopics, fetchEndPoints}
+function fetchArticleById(article_id) {
+    return db.query(`SELECT author, title, article_id, body, topic, created_at, votes, article_img_url
```

Comment:
not necessary, but could use async await here with db.query too if you wanted to keep that consistent through controllers and models